blocks/footer/footer.js

```diff
@@ -1,25 +1,21 @@
-import { readBlockConfig, decorateIcons } from '../../scripts/aem.js';
+import { getMetadata } from '../../scripts/aem.js';
+import { loadFragment } from '../fragment/fragment.js';
 
 /**
  * loads and decorates the footer
  * @param {Element} block The footer block element
  */
 export default async function decorate(block) {
-  const cfg = readBlockConfig(block);
+  const footerMeta = getMetadata('footer');
   block.textContent = '';
 
-  // fetch footer content
-  const footerPath = cfg.footer || '/footer';
-  const resp = await fetch(`${footerPath}.plain.html`, window.location.pathname.endsWith('/footer') ? { cache: 'reload' } : {});
+  // load footer fragment
+  const footerPath = footerMeta.footer || '/footer';
```

Comment:
@davidnuescheler I think this line should just be 

```
const footerPath = footerMeta || '/footer';
```

since switching to `getMetadata` instead of `readBlockConfig`. IIUC, `getMetadata` only returns a string so `footerMeta.footer` will always be undefined.

Is my assumption correct?